app/views/availabilities/show_everyone.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="container-fluid">
+  <h1>Schedule Availability for all users</h1>
+  <div class="scrollable">
+    <table class="table table-sm table-bordered table-availabilities-all">
+      <thead>
+      <tr>
+        <th class="availability-header-empty-cell"></th>
+        <% (1..7).each do |index| %>
+          <th><%= Availability.days.keys[index % 7][0..2] %></th>
+        <% end %>
+      </tr>
+      </thead>
+      <tbody>
+      <% @time_ranges.each do |time_range| %>
+        <tr>
+          <% rowspan = ((time_range.end_time - time_range.start_time) / 0.5.hours).round %>
+          <% if time_range.start_time.to_i.modulo(3600).zero? %>
+            <td rowspan="2"><%= time_range.start_time.strftime("%H%M") %></td>
+          <% end %>
+          <% (1..7).each do |index| %>
+            <td rowspan="<%= rowspan %>">
+              <ol class="availability-list">
+              <% @availabilities[[Availability.days.keys[index % 7], time_range.id]].each do |u| %>
+                <li><%= stylise_user(@users[u]) %></li>
+              <% end %>
+              </ol>
+            </td>
+          <% end %>
+        </tr>
+        <% if rowspan > 1 %>
```

Comment:
You actually don't need this if statement. Not sure if omitting is better though

Reply:
What do you mean no need if?

Reply:
From my testing, when all columns in a row have a `rowspan="2"`, I need to compensate for the skipped row else it would render right below (which because I have `rowspan="2"`, it's rendered overlapping with the row above it.) @donjar

Reply:
If `rowspan <= 1` then `rowspan - 1 <= 0` and `n.times` if `n <= 0` does not do anything.

Reply:
Lmao you're right I derped

Reply:
Done